api/graphql/schema.graphql

```diff
@@ -1962,6 +1972,11 @@ type Mutation {
     @update
     @guard
     @canFind(ability: "updatePublished", find: "id")
+  togglePoolUserBookmark(poolId: UUID! @rename(attribute: "pool_id")): Pool
+    @guard
+    @inject(context: "user.id", name: "user_id")
```

Comment:
You are injecting `user_id` here

Reply:
See [79cc264](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10509/commits/79cc264065ed10aee21f6ca7574f7031f338df8e)